nightwatch.conf.js

```diff
@@ -1,80 +0,0 @@
-// Helpful
```

Comment:
How come this file is being shown as deleted? I cloned your repo and the file is this there.

Reply:
@nodox I had 2 PR's so maybe you cloned from the first branch. On stellar_theme_branch I tried to remove everything aparat the 'src'. Sorry. Maybe I did it messy :)